app/src/main/java/com/compscieddy/meetinthemiddle/Util.java

```diff
@@ -40,17 +40,9 @@ public static Bitmap getCroppedBitmap(Context context, Bitmap bitmap) {
     return output;
   }
 
-  public static void rotateFabForward(View view) {
+  public static void rotateView(View view, float angle){
```

Comment:
@ambardhesi sure ok - that's fine then - to make it clear that's what you wanted to do then (make it more static and more of a helper), then could you move it to Utils